examples/local_noise_simulation.py

```diff
@@ -23,7 +23,7 @@
 
 
 circuit = Circuit().x(0).x(1)
-noise = Noise.BitFlip(probability=0.1)
```

Comment:
Why the noise change with this?

Reply:
Just wanted the entire file to be idiomatic